app/controllers/admin/accounts_controller.rb

```diff
@@ -0,0 +1,36 @@
+module Admin
+  class AccountsController < BaseController
+    load_and_authorize_resource
+
+    def index
+      @q = Account.includes(:registrar).search(params[:q])
```

Comment:
We are creating strong params on `account_params`, but using not protected `params` all the way?